libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/plugin/mpp/DefaultKotlinDependencyHandler.kt

```diff
@@ -107,4 +107,52 @@ class DefaultKotlinDependencyHandler(
 
     override fun npm(org: String?, packageName: String, version: String) =
         npm("${if (org != null) "@$org/" else ""}$packageName", version)
+
+    override fun devNpm(name: String, version: String): NpmDependency =
+        NpmDependency(
+            project = project,
+            name = name,
+            version = version,
+            scope = NpmDependency.Scope.DEV
+        )
+
+    override fun devNpm(name: String, directory: File): NpmDependency {
+        check(directory.isDirectory) {
```

Comment:
This check is duplicated with `npm` function
Can be extracted to separate function `directoryNpmDependency` which do this check and return `NpmDependency` with appropriate scope